### ISSUE TITLE
recurrent dropout missing in LSTM layer

### ISSUE THREAD

**Issue by partizanos** [calvin-zcx/SCEHR#1]
Hello and thank you a lot for this work. I am trying to recreate the "original" keras lstm of the benchmark. I see that keras has a recurrent_dropout argument that torch does not support . Is it compensated in some other part of the code (ie in pytorch_models/lstm.py ) ?  Thank you again for the very nice work
